examples/all-clusters-app/mbed/README.md

```diff
@@ -0,0 +1,237 @@
+device.
+
+It is possible to connect to external gdb-server session by using specific
+**'Flash Mbed (board name) remote'** task.
+
+## Debugging
+
+The All Clusters application can be debugging in the same way as any other
+Matter example ported to mbed-os platform.
+
+The [Open On-Chip Debugger](http://openocd.org/) is used to to open and manage
+the gdb-server session. Then gdb-client (arm-none-eabi-gdb) connect the server
+to upload binary image and control debugging.
+
+```
+Run and Debug (Ctrl+Shift+D) => Debug Mbed (board name) => Start Debugging (F5) => all-clusters-app => (build profile)
+```
+
+It is possible to connect to external gdb-server session by using specific
+**'Debug Mbed (board name) remote'** task.
+
+## Testing
+
+### CHIP Tools
+
+Read the [MbedCommissioning](../../../docs/guides/mbedos_commissioning.md) to
+see how to use different CHIP tools to commission and control the application
+within a WiFi network.
+
+## Supported devices
+
+| Manufacturer                                          | Hardware platform                                                         | Build target          | Platform image                                                                                                                                                                 |       Status       | Platform components                                                                                                                                                                                                                                                           |
+| ----------------------------------------------------- | ------------------------------------------------------------------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :----------------: | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [Cypress<br> Semiconductor](https://www.cypress.com/) | [CY8CPROTO-062-4343W](https://os.mbed.com/platforms/CY8CPROTO-062-4343W/) | `CY8CPROTO_062_4343W` | <details><summary>CY8CPROTO-062-4343W</summary><img src="https://os.mbed.com/media/cache/platforms/p6_wifi-bt_proto.png.250x250_q85.jpg" alt="CY8CPROTO-062-4343W"/></details> | :heavy_check_mark: | <details><summary>LEDs</summary><ul><li>Board has only one usable LED (LED4) which corresponds to USER LED from UI.</li></ul></details> <details><summary>Buttons</summary><ul><li>Unused</li></ul></details> <details><summary>Slider</summary><ul><li>Unused</ul></details> |
+
+There is a plan to expend above list with new platforms within mbed-os
```

Comment:
There is a plan to expand this list with new platforms...

(should we say "We plan to" or "Arm plans to"? who is planning this work? is the work actually planned, or just on a backlog - may be better not to commit Arm in writing if we aren't sure...)